jsonService/src/main/webapp/passfault.css

```diff
@@ -215,7 +215,6 @@ body {
 	clear: both;
 	padding: 1.5em;
 	max-width: 48em;
-	height: 100em;
```

Comment:
>  When scrolling down to see the (blue) sign it disappears before you can read it.
Fixed this now
>The "hover over" is really a bad experience anyway. It would be better to put an "info" icon in the corner of the sign, so clicking or tapping it would put a pop-up with the pattern information.
When clicking/tapping, the blue pattern appears at bottom.

Try this, let's see if you like it!